examples/orders_create_and_download_multiple_orders.py

```diff
@@ -50,33 +57,36 @@
                      [-117.558734, 45.229745]]]
 }
 
-oregon_images = ['20200909_182525_1014', '20200909_182524_1014']
 oregon_order = planet.order_request.build_request(
-    'oregon_order',
-    [planet.order_request.product(oregon_images, 'analytic_udm2', 'PSScene')],
-    tools=[planet.order_request.clip_tool(oregon_aoi)])
-
-
-async def create_and_download(order_detail, directory, client):
+    name='oregon_order',
+    products=[
+        planet.order_request.product(
+            item_ids=['20200909_182525_1014', '20200909_182524_1014'],
+            product_bundle='analytic_udm2',
+            item_type='PSScene')
+    ],
+    tools=[
+        planet.order_request.clip_tool(aoi=oregon_aoi)
+    ]
+)
+
+
+async def create_and_download(client, order_detail, directory):
+    """Place an order, wait for completion, and download assets as a single task.""" 
     with planet.reporting.StateBar(state='creating') as reporter:
-        # create
         order = await client.create_order(order_detail)
         reporter.update(state='created', order_id=order['id'])
-
-        # wait for completion
         await client.wait(order['id'], callback=reporter.update_state)
 
-    # download
```

Comment:
I removed obvious comments and rolled them up into a docstring.

Reply:
Let's do the same thing for the data example.